kivymd/uix/textfield.py

```diff
@@ -1116,7 +1117,7 @@ def on_disabled(self, *args):
             elif self.color_mode == "custom":
                 self._update_colors(self.line_color_focus)
 
-    def on_text(self, instance, text):
```

Comment:
I don't understand where you got this code from ... 
I fixed that a long time ago. And these changes are in the master branch.

![Uploading Снимок экрана 2021-02-16 в 18.35.15.png…]()

Reply:
To be clear about one thing - this PR was created against the current Master branch and when you look on GH at the "files changed" view or the "review view" you are seeing a diff against the Master Branch so you can see what the changes ARE against the Master branch.

This PR in the current form (after the conversation above about function names) is made up of two main changes:

- line 942 binds the `set_text` method to the `on_text` event.
- in lines 1127 - 1143 remove the `if self.focus` test since changes that come from binding the text attribute to a StringProperty are most likely to happen when the Widget is NOT focussed and this test means that this can leave the Widget in an incorrect state.

Which of those two changes were you referring to?